src/pages/TrueSightV2/components/TokenFilter/index.tsx

```diff
@@ -246,9 +246,9 @@ export default function TokenFilter({
               <StyledSelect
                 value={filter[queryKey]}
                 key={queryKey}
-                activeRender={item => <ActiveSelectItem name={displayName} label={item?.label} />}
+                activeRender={(item: any) => <ActiveSelectItem name={displayName} label={item?.label} />}
```

Comment:
I dont see any warning/error on this.
<img width="710" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/212bda66-64c5-4155-b7a6-3a8e5418e8d8">